backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -2080,15 +2077,14 @@ describe('ContributionResolver', () => {
           // stephen@hawking.uk: [1000, 1000, 1000] - deleted
           // garrick@ollivander.com: [1000, 1000, 1000] - not activated
 
-          describe('user for creation to update does not exist', () => {
+          describe.skip('user for creation to update does not exist', () => {
```

Comment:
why skip? If the user status is no criteria anymore lets remove the test?
Is there implications for the operation if the user was deleted/does not exist?